### DISCUSSION TITLE
腾讯云函数部署识别

### DISCUSSION THREAD

**Discussion by a145356** [luolongfei/freenom#49] [Q&A] [answered]
失败提示：未找到函数执行入口文件，请确认代码包入口文件名和 handler 设置是否对应或代码压缩包是否正常。

**Comment by a145356** [comment]
我已经解决了，我之前是下载压缩包解压，编辑环境变量文件，导致失败。我从新上传压缩包就OK了

**Comment by luolongfei** [comment]
环境变量可以直接在云函数那边编辑

**Comment by luolongfei** [comment, marked as answer]
压缩包只要确保index.php在根目录就行
